test/numbers.jl

```diff
@@ -2771,7 +2771,8 @@ let types = (Base.BitInteger_types..., BigInt, Bool,
     for S in types
         for op in (+, -)
             if S === Float16 # type instability here?
-                @test_broken @inferred Base.promote_op(op, S)
+                # broken, fixme then remove this branch
```

Comment:
... OK apparently `@inferred` is not even a test. In that case I'm fine with using `@test_throws` for now. Note that using this on a test will mess things up.

```jl
julia> @test_throws ErrorException @test false
Test Failed
  Expression: false
Test Passed
  Expression: @test false
      Thrown: ErrorException
```